cpp/src/barretenberg/benchmark/honk_bench/compare_honk_to_plonk_standard.sh

```diff
@@ -19,29 +19,23 @@ pip3 install -r $BUILD_DIR/_deps/benchmark-src/requirements.txt
 cd $BASE_DIR
 mkdir $BENCH_RESULTS_DIR
 
```

Comment:
Did you forget requirement installation here or is it not needed?

Reply:
Sorry, missed that it is an updated file with cut out lines.  Can add --user, though

Reply:
done